src/components/MultiSelect/helpers/normalizeValue.js

```diff
@@ -0,0 +1,26 @@
+export default function normalizeValue(value) {
+    if (!value || typeof value !== 'object') {
+        return [];
+    }
+    if (Array.isArray(value)) {
+        if (value.length === 0) {
+            return [];
+        }
+        return value.map(item => {
+            const { label, name, value: itemValue } = item;
+            return {
+                label,
+                name,
+                itemValue,
```

Comment:
why use itemValue here?
this means that the onChange will send this instead of value